codecarbon/emissions_tracker.py

```diff
@@ -49,6 +49,7 @@ def __init__(
         self._project_name: str = project_name
         self._measure_power_secs: int = measure_power_secs
         self._start_time: Optional[float] = None
+        self._last_measured_time: Optional[float] = None
```

Comment:
```suggestion
        self._last_measured_time: float = time.time()
```

See below for the reason behind this code change.